Sniffs.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Sniffs;
+
+class Sniffs
+{
+    const DEFAULT_POINTS = 30000;
```

Comment:
That's something I was going to ask for feedback on. The majority of the rules are *extremely* minor so I'm not really confident in how much we should accumulate based on these rules.